webmacs/main.py

```diff
@@ -236,9 +242,33 @@ def main():
                 if n.isdigit()]
         opts.instance = str(max(uniq) + 1) if uniq else "1"
 
-    conf_path = os.path.join(os.path.expanduser("~"), ".webmacs")
+    conf_path = opts.config
     if not os.path.isdir(conf_path):
-        os.makedirs(conf_path)
+        deprecated = os.path.expanduser("~/.webmacs")
+        if os.path.isdir(deprecated):
+            # Since logging has not been setup yet, use manual print
+            print("Warning: '{}' as an init directory has been deprecated. "
+                  "Please use '{}' or configure it using '-c' instead."
+                  .format(deprecated, conf_path), file=sys.stderr)
+            conf_path = deprecated
+        else:
+            os.makedirs(conf_path)
+
+    data_path = os.path.join(XDG_DATA_HOME, "webmacs")
+    if not os.path.isdir(data_path):
+        deprecated = os.path.expanduser("~/.webmacs")
+        if os.path.isdir(deprecated):
+            # Since logging has not been setup yet, use manual print
+            print("Warning: '{}' as a data directory has been deprecated. "
+                  "Please move your profile and adblock directories to '{}'."
```

Comment:
adblock directory is only used for cache, it should go in the xdg_cache_home too.

Reply:
Also, the current profile dir unfortunately contain a "cache" subdirectory which should not be copied in new data dir. Maybe you could rephrase to ask for moving profile/default instead of just cache?

Reply:
also, I started to use the warning module in some other places, such as https://github.com/parkouss/webmacs/blob/master/webmacs/keymaps/__init__.py#L545. Do you mind giving it a try instead of raw prints?

Reply:
Yup, I'll get these things fixed. Sorry, I didn't realize the warnings module even existed!

Reply:
Ah, no problem at all! It is not really a blocking issue but it would be more consistent.
I'll wait for your updates then. :)